opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml

```diff
@@ -276,6 +268,24 @@
          <parameter key="port" value="3389"/>
          <parameter key="timeout" value="3000"/>
       </service>
+      <downtime begin="0" end="300000" interval="30000"/><!-- 30s, 0, 5m -->
+      <downtime begin="300000" end="43200000" interval="300000"/><!-- 5m, 5m, 12h -->
+      <downtime begin="43200000" end="432000000" interval="600000"/><!-- 10m, 12h, 5d -->
+      <downtime begin="432000000" interval="3600000"/><!-- 1h, 5d -->
+   </package>
+
+   <!--
+      This package is defined for Minion nodes. Don't alter/remove this. If minions are not getting used, it is safe to remove below pkg.
+   -->
+   <package name="Minion">
+      <filter>foreignSource != 'Minions' AND IPADDR != '0.0.0.0'</filter>
```

Comment:
Shouldn't the filter be: `foreignSource == 'Minions' AND IPADDR != '0.0.0.0'`?

Reply:
my bad, Thanks for catching